src/hooks/useAccountData.ts

```diff
@@ -30,7 +30,7 @@ export const useAccountData = <T extends {}>(cli: MatrixClient, eventType: strin
     const handler = useCallback((event) => {
         if (event.getType() !== eventType) return;
         setValue(event.getContent());
-    }, [cli, eventType]);
+    }, [eventType]);
```

Comment:
I don't know enough about hooks, but this and the one below feel wrong.

Reply:
The array specifies which values have to change before the hook is re-triggered. In both cases neither hook's value depends on the removed argument.

Reply:
Its fine, I think its from before I changed how it worked

Reply:
👍